openfl-tutorials/experimental/Workflow_Interface_501_FineTuning_LLAMA2.ipynb

```diff
@@ -69,8 +69,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install git+https://github.com/intel/openfl.git\n",
-    "!pip install -r requirements_workflow_interface.txt"
+    "!pip install git+https://github.com/intel/openfl.git"
```

Comment:
Reminder to change this to `https://github.com/securefederatedai/openfl.git`

Reply:
Fixed